packages/dev/core/src/Physics/physicsEngineComponent.ts

```diff
@@ -34,7 +31,7 @@ declare module "../scene" {
          * @param plugin defines the physics engine to be used. defaults to CannonJS.
          * @returns a boolean indicating if the physics engine was initialized
          */
-        enablePhysics(gravity?: Nullable<Vector3>, plugin?: IPhysicsEnginePlugin): boolean;
+        enablePhysics(gravity?: Nullable<Vector3>, plugin?: any /*IPhysicsEnginePlugin*/): boolean;

```

Comment:
Same comment regarding this `any` - it can be more specific

Reply:
Fixed